pymc/sampling/forward.py

```diff
@@ -343,7 +362,7 @@ def sample_prior_predictive(
     var_names : Iterable[str]
         A list of names of variables for which to compute the prior predictive
         samples. Defaults to both observed and unobserved RVs. Transformed values
-        are not included unless explicitly defined in var_names.
+        are not allowed.
```

Comment:
Any reason why transformed (unconstrained) values are not allowed here? I need to get prior samples in the transformed space, what should I do? Thanks! 

(MCMC sampling by removing observations from the likelihood is not a good option. I guess I could also use `sample_prior_predictive` to get prior parameter samples and then transform back to the unconstrained space but it seems not easy to do it with `Transforms` in an efficient vectorized way, i.e., it seems I have to do a for-loop.)

Reply:
A helper to go from unconstrained back to constrained draws is needed elsewhere (specially since we don't save them in InferenceData after sampling), so we can add that and will also cover your use case.

I think there is an open issue for that. We can make it a blocker for these changes

Reply:
This issue: https://github.com/pymc-devs/pymc/issues/6721